src/components/lec6/lec6task7/gallery.js

```diff
@@ -0,0 +1,69 @@
+
+import React, { Component } from "react";
+import "./gallery.css";
+
+export default class Gallery extends Component {
+    constructor(props){
+        super(props);
+
+        this.state = {
+            posts: [],
+            isLoading: false,
+            error: null,
+            currentImg: 0,
+          };
+    }
+
+    componentDidMount() {
+        this.setState({ isLoading: true });
+
+        fetch('https://picsum.photos/v2/list?page=2&limit=1000')
+          .then(response => response.json())
+          .then(data => this.setState({ posts: data, isLoading: false }))
+          .catch(error => this.setState({ error, isLoading: false }));
+          console.log('componentDidMount');
+      }
+
+    prevImg = () => {
+        if (this.state.currentImg !==0)
+            this.setState({currentImg: --this.state.currentImg});
+        console.log('PrevImg', this.state.currentImg)
```

Comment:
Please use curly braces `{}` for your if statement, without it, you can get bugs in your code